package.json

```diff
@@ -3,14 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@syncfusion/ej2-react-dropdowns": "^20.1.51",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^13.5.0",
-    "axios": "^0.26.1",
```

Comment:
why did you delete Axios? Are you using another library that i am not seeing?

Reply:
my bad i hadn't downloaded yet. fixing it now.